importer.py

```diff
@@ -21,4 +21,37 @@
                              attachment_key='org.digitalpsych.redcap.data',
                              body=records)
     print(count)
+    try:
+        arr = LAMP.Type.get_attachment(researcher['id'], 'org.digitalpsych.redcap.links')["data"]
```

Comment:
We can do that, we'll just also need a list of the survey names as well.

Reply:
Maybe we make `IMPORT_SURVEY_LINKS` an array of strings? If the array is empty or the field is missing, skip this feature entirely. Otherwise, use that as the list of survey names.

Reply:
Yes, reading my mind. I just pushed that change. I'm going to update for the case with an empty array.

Reply:
This should be updated. Thanks for the suggestion.

Reply:
Awesome! Great work. 🎊